generateRandomQueries/README.md

```diff
@@ -0,0 +1,2 @@
+In order to generate a sql file that contains random procedures and calls, use `generate_procedure.py`. 
```

Comment:
minor: Maybe construct the README such that `procedure` is the first of the random query generation that we implement, and any other new ones can be added more naturally. So, start with a generic intro, and then have a section on `procedure`?